OxfordBodleian/Juel-Jensen/BDLaethb4.xml

```diff
@@ -1243,11 +1243,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     </list>
                                 </collation>
 
-                                <condition key="good">Some folia have water stains (e.g. <locus
-                                        from="41r" to="42v"/>, <locus from="108r" to="121r"/>). The
-                                    headband is broken, the tailband torn off. All edges were
-                                    trimmed; a note that was written in the outer upper margin of
-                                        <locus target="#4r"/> was cut.</condition>
+                                <condition key="good">
+                                  The manuscript is in a good state of preservation. It has only a few losses along the margins and deformations caused by humidity.
+                                  Some folia have water stains (e.g. <locus
+                                  from="41r" to="42v"/>, <locus from="108r" to="121r"/>). The
+                                  headband is broken, the tailband torn off. The leather cover is torn on the spine.<!--All edges were
+                                  trimmed; EDS: re-trimmed? It does not seem so to me --> <!-- a note that was written in the outer upper margin of
+                                  <locus target="#4r"/> was cut. EDS: I don't see it :( )--></condition>
```

Comment:
I do see black traces (not visible on the other pages) directly at the border of 4r, above the quire mark. Hard to say if it was writing or a smudge.
(re-)trimming will require some attention from us all at the next revision step. I think this has not been noted in all manuscripts and is somewhat controversial. I'd say that for now we can keep it in the comments and will come back to this point later.

Reply:
You are right, in the inner margin of f. 4r there are black traces but I've interpreted them as insect droppings since the manuscript doesn't seem to have been rebound...but yeah, it is difficult to say with certainty! 
For sure in the revision we should have a look at the re-trimming. In the last binding descriptions I have kept an eye on it, but it needs to be checked systematically.